app/queries/get_change_offer_options.rb

```diff
@@ -48,6 +48,9 @@ def offerable_courses
     make_decisions_courses
     .where(recruitment_cycle_year: recruitment_cycle_year)
     .where(ratifying_provider_is_preserved)
+    .joins(:course_options)
```

Comment:
What will happen if a candidate applied to a course option which then got marked as not valid, will this stop providers from making/changing an offer from the current course selected?

Reply:
I tested this manually and there doesn't seem to be anything stopping a provider making an offer for a course that the candidate selected even if the site became invalid. The provider cannot change course to a site that is now invalid, so I guess if they change the course once and then want to change it back to the one the candidate originally selected they _might_ not be able to do that.

I guess the alternative to this, if we want to give providers the ability to select any course option regardless of site being valid, would be to just relax the rules for which sites to display (if we remove this line then the change in this PR becomes unecessay https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/queries/get_change_offer_options.rb#L31).

Reply:
Good point, we did relax the rules because of all the support requests, let me enquire around this and get back to you around also removing this restriction

Reply:
Hi @stevehook, I've got confirmation that we can drop the `selectable` scope on course option meaning we won't need this change anymore. You happy to do it in this PR or for me to open a new one instead?

Reply:
Realised you are on holiday, the current PR can be closed in favour of: https://github.com/DFE-Digital/apply-for-teacher-training/pull/5288

Reply:
Thanks @cwrw, closing this one now...